source/slang/slang-dxc-support.cpp

```diff
@@ -46,8 +46,8 @@ namespace Slang
     }
 
     SlangResult emitDXILForEntryPointUsingDXC(
+        ComponentType*          program,
         BackEndCompileRequest*  compileRequest,
```

Comment:
I believe that we can technically extract the `ComponentType` that represents the program from the `BackEndCompileRequest` rather than have to pass it down, but this is fine in the name of being more explicit.